hipogram/posts/admin.py

```diff
@@ -1,5 +1,18 @@
 from django.contrib import admin
+from .models import Post, Tag
 
-from .models import Post
+admin.site.register(Tag)
 
-admin.site.register(Post)
+
+class PostAdmin(admin.ModelAdmin):
+    list_display = ("created_by", "creation_datetime", "Tags")
+    list_filter = ("created_by", "tags")
+
+    def Tags(self, obj):
+        queryset = []
```

Comment:
`queryset` naming is misleading IMO, it's not actually a queryset, it's just a list of objects. As a code styling advice, you can make this method a one-liner by using list comprehension or the `join` method of `String`.

Reply:
Do you mean something like:
`def tags(self, post):
return post.tags.all()
`
When I did this, the output in the admin panel was ugly.

Reply:
No, not this. First, when I said one-liner, I meant one-liner return, not including `def tags` part. Something like that:
```python
def _tags(self, post):
    return a nice list or string
````

Reply:
If you are not familiar with list comprehension and string join terms, please see:

https://docs.python.org/3/tutorial/datastructures.html#list-comprehensions
https://www.kite.com/python/answers/how-to-join-a-list-of-strings-in-python

Reply:
I know you meant that. I guess I forgot to press enter in my version. I'm working on the other matters. Thanks.